dist/GaugeChart/index.js

```diff
@@ -11,7 +11,7 @@ var _d = require("d3");
 
 var _propTypes = _interopRequireDefault(require("prop-types"));
 
-require("./style.css");
+// require("./style.css");
```

Comment:
Why did you remove this? Won't that affect the styling?

Reply:
I removed it since nextjs restricts css imports in modules. As I tested it, it does not affect styling.
However, to solve this and ensure that the styles are not tempered with, include the css in the index.js file as CSS in JS. Allow me to update the pull request.

Reply:
Added style in src/lib/GaugeChart/index.js